src/Board.py

```diff
@@ -212,23 +219,37 @@ def makeUnicodeStringRep(self, pieces: list[Piece]) -> str:
                     if p.position == C(x, y):
                         piece = p
                         break
-                bg_color = bg(self.tileColors[(x+y)%2]) if self.isCheckered else ''
+                bg_color = (
+                    bg(self.tileColors[(x + y) % 2])
+                    if self.isCheckered
+                    else ''
+                )
                 pieceRep = bg_color + '  ' + attr(0)
                 if piece:
                     side = piece.side
-                    color = self.whiteColor if side == WHITE else self.blackColor
+                    color = (
+                        self.whiteColor if side == WHITE else self.blackColor
+                    )
                     fg_color = fg(color)
-                    pieceRep = fg_color + bg_color + DISPLAY_LOOKUP[piece.stringRep] + ' ' + attr(0)
+                    pieceRep = (
+                            fg_color
+                            + bg_color
+                            + DISPLAY_LOOKUP[piece.stringRep]
+                            + ' '
+                            + attr(0)
```

Comment:
This only need to be indented once, not two times.

Reply:
Like that?
```
                    pieceRep = fg_color + bg_color + DISPLAY_LOOKUP[
                        piece.stringRep] + ' ' + attr(0)
```